doc/source/user_guide/indexing.rst

```diff
@@ -6,17 +6,18 @@
 Indexing and Selecting Data
 ***************************
 
-The axis labeling information in pandas objects serves many purposes:
+In pandas, data structure axes such as the rows and columns of a DataFrame may have
```

Comment:
To be clear, an Index (which is used for selection along an axis) *does* have labels. This is a main thing that separates pandas from libraries like NumPy

Reply:
Yes I know. It's actually quite challenging to unpack axis, index, labels, positional indexation via integers. An index really *is* labels. I found it an improvement over "axis labeling information ... serves many purposes", by being less nebulous and vague.